cases/services/tag_enricher.py

```
@@ -773,20 +775,20 @@ def _collect_evidence_text(case: Case) -> str:  # noqa
         return " ".join(parts)
 
     try:
-        all_sources = DocumentSource.objects.filter(source_id__in=source_ids)
-        high_value = all_sources.filter(source_type__in=HIGH_VALUE_SOURCE_TYPES)
-        other = all_sources.exclude(source_type__in=HIGH_VALUE_SOURCE_TYPES)
+        all_sources = list(
+            DocumentSource.objects.filter(source_id__in=source_ids)
+        )
     except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)